### PR TITLE
Correct cron expression: it has granularity up to the second, so the …

### PR DESCRIPTION
**Pull request by kajahno**
…previous expression was wrong

I've also noticed that the deployment path was wrong in the 'deploy' script, so also fixed that.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -8,7 +8,7 @@
     "start": "node app.js",
     "build": "babel . --ignore node_modules,es5-packaged,pupeteer --out-dir es5-packaged",
     "package": "npx pkg -t node16-linuxstatic-x64,node16-win-x64 -o elprimobot es5-packaged/app.js",
-    "deploy": "rsync  -avz elprimobot-linuxstatic elprimobot:~/development/",
+    "deploy": "rsync  -avz elprimobot-linuxstatic elprimobot:~/elprimobot/",
```
Comment (whoan) [inline package.json:11]: Was this change intended to be here? Feels unrelated.

File: app.js [hunk 2]
```diff
@@ -32,7 +32,7 @@ const activeGames = {};
  */
 initializeDiscordClient();
 const job = new CronJob(
-    '30 0 * * * *',
+    '30 0 0 * * *',
```
Comment (whoan) [inline app.js:35]: LG according to the doc: https://github.com/kelektiv/node-cron#cron-ranges
Why wasn't it happening before?

File: package.json (hunk 1 above)
Comment (kajahno) [reply package.json:11]: Thanks I've noticed the deployment path was wrong, and sneakily fixed it :D will add to the PR description

File: app.js (hunk 2 above)
Comment (kajahno) [reply app.js:35]: I missed that the precision was 'seconds', so basically:
* before: it was running every minute 0, of every hour, on the second 30 (once every hour)
* after: it was running every minute 0, of every hour 0, on the second 30 (once per day)
